code/modules/food_and_drinks/food/snacks_meat.dm

```diff
@@ -184,6 +184,9 @@
 	foodtype = MEAT | SUGAR
 
 /obj/item/reagent_containers/food/snacks/monkeycube/proc/Expand()
+	if(CONFIG_GET(number/monkeycap) <= GLOB.monkeys.len)
+		visible_message("<span class='warning'>[src] gives a single shudder, before drying out.</span>")
```

Comment:
There should be some indication as to why, even if it wont make perfect sense in universe